### ISSUE TITLE
Table footer not appearing with version 1.7.0

### ISSUE THREAD

**Issue by skegel13** [spatie/vue-table-component#81]
I have confirmed I am using version 1.7.0 and I am trying to use the tfoot slot but it does not appear to be working.

Inside my table component, I am using:
```
<template slot="tfoot" scope="{ rows }">
    <tr>
        <th>Totals:</th>
	<th>{{ rows.reduce((sum, value) => { return sum + value.data.initial_amount; }, 0) }}</th>
	<th>{{ rows.reduce((sum, value) => { return sum + value.data.balance; }, 0) }}</th>
	<th>{{ rows.reduce((sum, value) => { return sum + value.data.remaining; }, 0) }}</th>
	<th>{{ rows.reduce((sum, value) => { return sum + value.data.payment; }, 0) }}</th>
    </tr>
</template>
```
I don't have any kind of tfoot tags in the dom either.

**Comment by GreggOrdHume** [comment]
@skegel13 - I am going to be needing a footer too. Let me try it this week and if I get it right I will let you know.

**Comment by radel** [comment]
From what I understand, the building process when installing the package is broken, so in then dist directory there is no trace of the tfoot slot. 

Importing the original index.js makes tfoot slot available:
import { TableComponent, TableColumn } from "vue-table-component/src/index.js"

Webpack is emitting a minor warning though with the Vue version I'm using, I will report it as a new issue.

**Comment by GreggOrdHume** [comment]
For me when doing the above:
`import { TableComponent, TableColumn } from "vue-table-component/src/index.js"`

I get the following errors with Webpack, any ideas?
```
ERROR in ./node_modules/vue-table-component/src/index.js
Module not found: Error: Can't resolve './components/Pagination' in 'node_modules/vue-table-component/src'
 @ ./node_modules/vue-table-component/src/index.js 3:0-49
 @ ./local/js/pages/ticket.js

ERROR in ./node_modules/vue-table-component/src/index.js
Module not found: Error: Can't resolve './components/TableColumn' in 'node_modules/vue-table-component/src'
 @ ./node_modules/vue-table-component/src/index.js 2:0-51
 @ ./local/js/pages/ticket.js

ERROR in ./node_modules/vue-table-component/src/index.js
Module not found: Error: Can't resolve './components/TableComponent' in 'node_modules/vue-table-component/src'
 @ ./node_modules/vue-table-component/src/index.js 1:0-57
 @ ./local/js/pages/ticket.js
```

**Comment by sebastiandedeyne** [comment]
Hmm looks like something went wrong with publishing indeed. Just released 1.7.1, should resolve this issue!
